README.md

```diff
@@ -3,9 +3,9 @@ rally-tracks
 
 This repository contains the default track specifications for the Elasticsearch benchmarking tool [Rally](https://github.com/elastic/rally).
 
-Tracks are used to describe benchmarks in Rally.
+Tracks are used to describe benchmarks in Rally. For each track, the README.md file documents the data used, explains its parameters and provides an example document.
```

Comment:
I am not sure we can say that that all of them have one e.g. the `metricbeat/` one (which is hasn't been updated for some time).

Reply:
Well, it's the only one missing a description, not sure if that should block this PR?

Reply:
> Well, it's the only one missing a description, not sure if that should block this PR?

Nope. I left a comment instead of an LGTM by accident.